spec/classes/pulpcore_spec.rb

```diff
@@ -513,6 +513,17 @@
             .with_content(%r{\s'level': 'DEBUG',})
         end
       end
+
+      context 'with parameter worker_ttl = 60' do
+        let :params do
+          { worker_ttl: 60 }
+        end
+
+        it 'configures pulpcore with setting WORKER_TTL = 60' do
+          is_expected.to contain_concat__fragment('base')
+            .with_content(%r{^WORKER_TTL = 60$})
```

Comment:
Please add the opposite of this to the base test. So around line 29 you should add `.without_content(%r{WORKER_TTL})`